modules/library/metadata/src/main/java/org/geotools/measure/UnitDefinitions.java

```diff
@@ -0,0 +1,369 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.measure;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.unmodifiableList;
+
+import java.util.List;
+import javax.measure.MetricPrefix;
+import si.uom.NonSI;
+import si.uom.SI;
+import systems.uom.common.USCustomary;
+import tech.units.indriya.AbstractUnit;
+import tech.units.indriya.unit.Units;
+
+public final class UnitDefinitions {
```

Comment:
@aaime The AOSP formatting does a really poor job in this file. :-/

Reply:
The AOSP formatting has been voted by the Project Management Committee, you can argue about it on the devel list, and if approved, do the work necessary to change the formatting for the whole GeoTools. 
On a more practical note, it's typically easy to change a bit the sources so that the formatting produces a nicer output (e.g, reduce nesting, not sure what is happening here, don't have time to check now).

Reply:
Just wanted to note that the formatting results weren't my choice, I have no claims beyond that.